src/centos/macros.tpl

```diff
@@ -41,6 +41,19 @@ RUN \
 {% else %}
     virtualenv-$PYTHON_VERSION ${APP_ROOT} && \
 {% endif %}
+{% if spec.version != "2.7" %}
+    # Python 3 only code, Python 2 installs pip from PyPI in the assemble script. \
+    # We have to upgrade pip to a newer verison because: \
+    # * pip < 9 does not support different packages' versions for Python 2/3 \
+    # * pip < 19.3 does not support manylinux2014 wheels. Only manylinux2014 wheels \
+    #   support platforms like ppc64le, aarch64 or armv7 \
+    # We are newly using wheel from one of the latest stable Fedora releases (from RPM python-pip-wheel) \
+    # because it's tested better that whatever version from PyPI and contains useful patches. \
```

Comment:
```suggestion
    # because it's tested better that newer versions from PyPI and contains useful patches. \
```

Reply:
I'd say that the Fedora wheel is also tested better than older versions from PyPI. Isn't it? It's only comment, nothing really visible to users.

Reply:
Oh, I get what you mean. In that case there's a typo: `that` instead of `then`.

Reply:
Fixed.